src/uncontrast/pages/EffectsDistributionLines/EffectsDistributionLines.jsx

```diff
@@ -122,7 +173,7 @@ export default function EffectsDistributionLines() {
 
     queryParams.get("bin_size")
```

Comment:
This is not clear maybe extract the value like

`const binSizeFromParams =queryParams.get("bin_size")
if (binSizeFromParams) {
...
}
 `